cmd/main.go

```diff
@@ -27,6 +27,7 @@ import (
 
 func main() {
 	c, err := cli.New(
+		cli.WithVersion(version.String()),
```

Comment:
what is the use case for it? Why someone would like to run for example `mybin version` and get a result `kubeBuilderVersion` instead of the version of the bin?

Reply:
`mybin version` is contructed by a different `cli.New` call which would have the string they want.

```
c, err := cli.New(
	cli.WithCommandName("mybin"),
	cli.WithVersion("v0"),
	...
)
```

Reply:
but the output will no be `kubeBuilderVersion:v0`? could you please add here its output for we check?

Reply:
`version.String()` outputs all the `kubebuilder` info, but `mybin` would not use `version.String()`